### PR TITLE
update changelog for messaging and iid

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Firebase/InstanceID/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 2020-06 -- 4.4.0
+# [changed] Standardize FirebaseCore import headers. (#5758)
```
Comment (paulb777) [inline Firebase/InstanceID/CHANGELOG.md:2]: I'll leave to @morganchen12  about how best to centralize or de-centralize this for public consumption, but it's fine to stay in CHANGELOG even if it doesn't go out in the release notes.

Comment (chliangGoogle) [reply Firebase/InstanceID/CHANGELOG.md:2]: it's the only change for iid so we need an entry here but i'm open for other wording.

Comment (morganchen12) [reply Firebase/InstanceID/CHANGELOG.md:2]: Given the thorough explanation in the core changelog, I'm ok with making this as succinct as possible. This should be a list item, though, not a header. (Replace `#` with `-`)

Comment (chliangGoogle) [reply Firebase/InstanceID/CHANGELOG.md]: Done.

**Pull request merged**